OpenSourcePolitics/decidim-app
# Fix display condition answer option in templates

#### :tophat: Description
When in a Questionnaire you create a Question with a Display Condition, you select a question that have 2 choices (Matrix Single option or Multiple options) and set the condition "Equal to" the dropdown list of the Answer option is empty.

#### :pushpin: Related Issues
*Link your PR to an issue*
- [Notion card](https://www.notion.so/opensourcepolitics/bd4bf860eab94f9daca7da803f09e3b2?v=c96df65755dd403d95e86f82d7749709&p=f166b4744d684e22b5481332b8e67743&pm=c)

#### Testing

1. Create a Questionnaire Template
2. Create a first mandatory question of Matrix Type Single option with 2 answer choices (cf screenshot one)
3. Create a 2nd Question with a display condition.
4. For the condition question, select the 1st question and then for the condition select "Equal to"
5. Click on the Answer option dropdown list
6. See that the dropdown list contains the 2 answer options of the first question (cf screenshot two)

#### Tasks
- [X ] Add specs

### :camera: Screenshots
ONE => 
<img width="1020" alt="one" src="https://github.com/user-attachments/assets/fb72f4f3-4b81-480c-be09-fa47b141f3cc">

TWO =>
<img width="1004" alt="two" src="https://github.com/user-attachments/assets/f34eeac5-3605-4f90-8a1c-7fca35658085">
